app/views/session_timeout/_warning.html.erb

```diff
@@ -9,6 +9,9 @@
         <p id='<%= description_id %>'>
           <%= modal.message %>
         </p>
+        <p class="usa-sr-only" role="timer" aria-live="polite" aria-atomic="true">
```

Comment:
My read of the `timer` role is that it should only contain a numeric value:

>A type of [live region](https://www.w3.org/TR/wai-aria-1.1/#dfn-live-region) containing a numerical counter which indicates an amount of elapsed time from a start point, or the time remaining until an end point.

https://www.w3.org/TR/wai-aria-1.1/#timer

Put another way: Is there any behavior of the `timer` role we're hoping to achieve which isn't otherwise provided by the `aria-live` behavior?

Alternatively, with the `timer` being a subclass of [the `status` role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_status_role), maybe the latter could be appropriate here, since it's designed to allow for more general text content?

Reply:
The goal of this change is for screen reader and accessibility support and the aria role of timer is for exactly this purpose, so I'm hoping this will work better with assistive devices across the board. As a note, with this configuration, screen readers add the words `minutes`, `seconds`, etc. when reading the number.

Reply:
Ok, I've updated it to go with `XX minutes and XX seconds` format for visual notification and `XX:XX:XX` for screen readers.